datalad/distribution/tests/test_install.py

```diff
@@ -225,7 +225,7 @@ def test_install_simple_local(src, path):
     # installing it again, shouldn't matter:
     with swallow_logs(new_level=logging.INFO) as cml:
         ds = install(src, path=path)
-        cml.assert_logged(msg="{0} appears to be installed already.".format(ds),
+        cml.assert_logged(msg="{0} was already installed from".format(ds),
                           regex=False, level="INFO")
```

Comment:
for paranoid me, would you be so kind to test that annex-uuid stays the same? smth like

```python
old_uuid = AnnexRepo(path).uuid
ds = install(src, path=path)
eq_(old_uuid, AnnexRepo(path).uuid)
```
to assure that annex was not reinitialized (if this ds is actually an annex one)